src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/InitialConsiderationController.java

```diff
@@ -35,12 +35,12 @@ public class InitialConsiderationController {
     private final VerifyTokenService verifyTokenService;
     private final InitialConsiderationService initialConsiderationService;
     private static final String INVALID_TOKEN = "Invalid Token {}";
-    private static final String COMPLETE_IC_BODY = "<hr>"
-        + "<h3>What happens next</h3>"
-        + "<p>A tribunal caseworker will act on any instructions set out in your initial consideration to progress "
-        + "the case. "
-        + "You can <a href=\"/cases/case-details/${[CASE_REFERENCE]}#Documents\" target=\"_blank\">view the initial "
-        + "consideration document in the Documents tab (opens in new tab).</a></p>";
+    private static final String COMPLETE_IC_HDR = "<h1>Initial consideration complete</h1>";
+    private static final String COMPLETE_IC_BODY = "<hr>" +
+        "<h3>What happens next</h3>" +
+        "<p>A tribunal caseworker will act on any instructions set out in your initial consideration to progress the case. " +
+        "You can <a href=\"/cases/case-details/${[CASE_REFERENCE]}#Documents\" target=\"_blank\">view the initial " +
+        "consideration document in the Documents tab (opens in new tab).</a></p>";
```

Comment:
The are some checkstyle issues in these files. '+' should be at the start of lines rather than at the end. One of these lines (41) goes over the 120 char limit, check it but I think just moving the '+' might fix it

Reply:
Fixed